app/src/js/components/Operations/overview.js

```diff
@@ -40,10 +40,10 @@ const tableHeader = [
 ];
 
 const statusOptions = {
-  Running: 'running',
-  Succeeded: 'succeeded',
-  Task_failed: 'task_failed',
-  Runner_failed: 'runner_failed'
+  Running: 'RUNNING',
+  Succeeded: 'SUCCEEDED',
+  Task_failed: 'TASK_FAILED',
+  Runner_failed: 'RUNNER_FAILED'
```

Comment:
We should also change these two to be 'Task Failed' and 'Runner Failed' so that is what shows up in the dropdown instead of the underbarred.

Reply:
Yup, I agree. Updated